### PR TITLE
[Merged by Bors] - fix: silence `linter.minImports` tests

### PR DESCRIPTION
**Pull request by adomani**
Reported on [Zulip](https://leanprover.zulipchat.com/#narrow/channel/428973-nightly-testing/topic/.2318333.20adaptations.20for.20nightly-2024-10-28)

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by kim-em** [comment]
Oh, this is really hacky. :-( It's going to fail as soon as we rename `test` to `MathlibTest`.

Could we have an option instead? Sorry this is turning out to be complicated. :-(

**Comment by adomani** [comment]
Ok, I agree that the earlier version was hacky!  I updated it with a separate option to take care of discriminating the two modes.

**Review by Vierkantor — APPROVED** [review]
LGTM otherwise!

bors d+

File: Mathlib/Tactic/Linter/MinImports.lean [hunk 1]
```diff
@@ -128,8 +137,14 @@ def minImportsLinter : Linter where run := withSetOptionIn fun stx ↦ do
...
       let redundant := importsSoFar.toArray.filter (!currImports.contains ·)
+      -- to make `test` files more stable, we suppress the exact count of import changes if
+      -- the `linter.minImports.increases` option is `false`
+      let count :=  if Linter.getLinterValue linter.minImports.increases (← getOptions) then
+                      m!"{newCumulImps - oldCumulImps}"
+                    else
+                      m!"X"
       Linter.logLint linter.minImports stx <|
-        m!"Imports increased by {newCumulImps - oldCumulImps} to\n{currImpArray}\n\n\
+        m!"Imports increased by {count} to\n{currImpArray}\n\n\
```
[the remark this answers is not in the document]
Comment (adomani) [reply Mathlib/Tactic/Linter/MinImports.lean]: Thanks, I like this suggestion!

**Comment by adomani** [comment]
bors merge

**Pull request closed without merging**